pkg/skaffold/deploy/labeller.go

```diff
@@ -19,40 +19,50 @@ package deploy
 import (
 	"fmt"
 
+	"github.com/google/uuid"
+
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/version"
 )
 
 const (
-	K8ManagedByLabelKey = "app.kubernetes.io/managed-by"
-	UnknownVersion      = "unknown"
-	Empty               = ""
+	K8sManagedByLabelKey = "app.kubernetes.io/managed-by"
+	UuidRunIdLabel       = "skaffold.dev/run-id"
```

Comment:
```suggestion
	UUIDRunIdLabel       = "skaffold.dev/run-id"
```
go style suggests having acronyms always spelled with all capitals

Reply:
hah, the linter caught it too! :D

Reply:
:man_facepalming: Of course!